README.md

```diff
@@ -983,6 +984,105 @@ This section describes code style for [ECMAScript 2015 Language Specification](h
 
 [⬆ back to TOC](#table-of-contents)
 
+### Arrow functions
+
+* [Arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+  should be used where an anonymous function is expected (when you need function and you don't want bind it to an
+  identifier):
+
+  > Explanation:
+  > * `this` keyword in arrow function binds to the upper scope's `this`. It prevents usage invalid `this` or usage less
+  >    efficient `Function.prototype.bind` function.
+  > * The notation is shorter than function expression.
+
+  ```js
+  [1, 2, 3].map((x) => {
+      // ...
+  });
+
+  // Use `function` here, because pass `someObj` as `this` argument.
+  [1, 2, 3].map(function (x) {
+      // ...
+  }, someObj);
+  ```
+
+* If you bind function with some name, use `function declaration`:
+
+  **Good:**
+
+  ```js
+  [1, 2, 3].filter(isGoodEnough);
+
+  function isGoodEnough(x) {}
+  ```
+
+  **Bad:**
+
+  ```js
+  const isGoodEnough = (x) => {};
+
+  [1, 2, 3].filter(isGoodEnough);
+  ```
+
+* If you export function, use `function declaration` or `function expression`:
+
+  **Good:**
+
+  ```js
+  export function foo() {}
+
+  export default function () {}
+  ```
+
+  **Bad:**
+
+  ```js
+  export foo = () => {};
+
+  export default () => {}
+  ```
+
+* Always add parentheses around arrow function parameters:
+
+  > Explanation:
+  > * This style is consistent with cases when function takes zero or more than one parameters.
+  > * You less typing when number of function parameters is changed.
```

Comment:
...You need to alter the code less, if the number of parameters the function accepts changes...